src/Details.js

```diff
@@ -1,8 +1,25 @@
 import React from 'react'
+class app extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+        message: `Hello, this will be the details page 
```

Comment:
Please indent these 2 lines. I also wouldn't use back-ticks here, as they will probably introduce an actual break line on the UI (`page\n...`)